mqtt/mqtt-bridge/src/bridge.rs

```diff
@@ -16,6 +18,18 @@ use crate::{
 
 const BATCH_SIZE: usize = 10;
 
+#[derive(Debug, PartialEq)]
+pub enum BridgeMessage {
```

Comment:
this is the message to pump, right? should we call it PumpMessage instead?

Reply:
addressed in master PR